tests/sentry/backup/__init__.py

```diff
@@ -95,3 +97,17 @@ def wrapped(*args, **kwargs):
         return wrapped
 
     return decorator
+
+
+def run_backup_tests_only_on_single_db(test_case: Type | Callable[..., Any]):
+    """Skip tests on backup functionality if testing on a split DB.
+
+    The backup commands currently are implemented only for a monolithic database.
+    When backup eventually supports importing and exporting JSON for databases in
+    individual Hybrid Cloud silos, update the affected tests and remove all instances
+    of this decorator.
+    """
```

Comment:
@azaslavsky Could you verify that this docstring accurately reflects the current state of the backup implementation and our future plans/hopes for it?

Feel free to add anything else that would be helpful to capture.

Reply:
Do you mind adding the comment below somewhere here (either in docstring or a regulard comment nearby) so that I can grep for it later when implementing?

```
# TODO(getsentry/team-ospo#185
```